apps/nav-veileders-flate/src/components/tiltak/DeltakelsesmengdeInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import { BodyShort, Heading } from '@navikt/ds-react'
+import { PameldingResponse } from '../../api/data/pamelding'
+import { deltakerprosentText, formatDate } from 'deltaker-flate-common'
+
+interface Props {
+  deltaker: PameldingResponse
+}
+
+export function DeltakelsesmengdeInfo({ deltaker }: Props) {
```

Comment:
Burde denne også vært for innbygger? så den kan lages i common?

Reply:
Sant, glemte den 👍